tools/datasets-info-create.py

```diff
@@ -138,7 +138,7 @@ def query(key, comment, multi_line=False, multi_item=False, extra=None):
         helper_info = json.load(json_data)
 
 info["title"] = query("title", "Title is composed of one or few words. Use sentence case.")
-info["year"] = query("year", "Year of first publication of the data set.")
+info["year"] = int(query("year", "Year of first publication of the data set."))
```

Comment:
Can you check that adding data sets with unknown years is also possible and supported.
I am guessing that at the query, the user would just press enter and leave the field blank.
If this causes a ValueError with current code, you could change to something like:
```
year = query(...)
info["year"] = int(year) if year else None
```
(needs to be tested)